container-images/tcib/base/os/tobiko/tobiko.yaml

```diff
@@ -0,0 +1,35 @@
+tcib_actions:
+- run: bash /usr/local/bin/uid_gid_manage {{ tcib_user }}
+- run: dnf -y install {{ tcib_packages.common | join(' ') }} && dnf clean all && rm -rf /var/cache/dnf
+- run: cp /usr/share/tcib/container-images/tcib/base/os/tobiko/tobiko_sudoers /etc/sudoers.d/tobiko_sudoers
+- run: chmod 440 /etc/sudoers.d/tobiko_sudoers
+- run: cd /var/lib; git clone https://opendev.org/x/tobiko
```

Comment:
https://logserver.rdoproject.org/77/77/09024d7e35356346fed263be307cddb54283f66f/github-check/tcib-build-containers/fd751d1/ci-framework-data/logs/3cff3b97-e7e8-49be-8802-9b127dc1c7b4/base/os/tobiko/tobiko-build.lofg

```
fatal: destination path 'tobiko' already exists and is not an empty directory.
```
https://github.com/openstack-k8s-operators/tcib/pull/77/files#diff-446dce5a18b6ebc4ffb5467990ee5dd4c0f052c7518793cf490f2b5def1eefc2R72 add the tobiko directory. We might need a different directory to keep the tobiko code.

Reply:
Thank you for the review. I have just changed the path of the tobiko repo code.